Example/Core/Tests/FIRAppAssociationRegistrationUnitTests.m

```diff
@@ -52,8 +50,8 @@ - (void)testPassObject {
   id result = [FIRAppAssociationRegistration registeredObjectWithHost:host
                                                                   key:kKey
                                                         creationBlock:^id _Nullable() {
-    return obj;
-  }];
+                                                          return obj;
```

Comment:
While I completely see your point of view, the flip side of things is that while it's not perfect it is completely automated and you no longer have to think about aligning these things, nor will you ever have to argue about formatting in a review again :-).

I started in your position as well, but over the last year and a bit on a project that used clang-format, I've come around: as long as the formatting is good enough, being able to delegate this tedium to a tool saves a lot of time and discussion.